modules/client-configuration/pages/vhm-caasp.adoc

```diff
@@ -57,3 +58,446 @@ When using Salt or {productname} (either via UI or API) on any {caasp} nodes:
+
+
+
+.Procedure: Electing a Management Node
+. In the {productname} {webui}, navigate to menu:Systems[System List] and click the name of the client to elect as the management node.
+. Navigate to the menu:Formulas[Configuration] tab, and check the ``CaaSP Management Node`` formula.
+. Click btn:[Save] and apply the highstate.
+
+
+[NOTE]
+====
+You will not be able to use the management node until the highstate has been completed.
+====
+
+
+List all known clusters by navigating to menu:Clusters[Overview].
+This list displays all existing clusters, along with the cluster type, and which management node they are associated with.
+It also shows the nodes within the cluster, if the nodes are registered to {productname}.
+For the nodes within a cluster, additional information from ``skuba`` and the {k8s} API are shown, including the role, status, and whether any updates are available.
+
+For more information about the data available for nodes, see https://documentation.suse.com/suse-caasp/4/html/caasp-admin/_cluster_updates.html.
+
+
+
+=== Manage Clusters
+
+To manage a cluster in {productname}, add the cluster in the {webui}.
+
+
+
+.Procedure: Adding an Existing Cluster
+. In the {productname} {webui}, navigate to menu:Clusters[Overview] and click btn:[FIXME].
+. Follow the prompts to provide information about your cluster, including the cluster type, and select the management node to associate.
+. Type the path to the ``skuba`` configuration file for the cluster.
+// For example?
+. Type the passwordless SSH key you want to use, or to the ``ssh-agent`` socket.
+. Type a name, label, and description for the cluster.
+. Click btn:[FIXME].
+
+
+For each cluster you manage with {productname}, a corresponding system group is created.
+By default, the system group is called ``Cluster <cluster_name>``.
+Refresh the system group to update the list of nodes.
+Only nodes known to {productname} are shown.
+
+
+You can remove clusters from {productname} by navigating to menu:Clusters[Overview], unchecking the cluster to be deleted, and clicking btn:[Delete Cluster].
+
+
+[IMPORTANT]
+====
+Deleting a cluster removes the cluster from {productname}, it does not delete the cluster nodes.
+Workloads running on the cluster will continue uninterrupted.
+====
+
+
+
+=== Manage Nodes
+
+When you have the cluster created in {productname}, you can manage nodes within the cluster.
+
+Before you add a new node to the cluster, check the management node can access the node you want to add using passwordless SSH, or the ``ssh-agent`` socket you are forwarding.
+
+You also need to ensure that the node you want to add is registered to {productname}, and has a {caasp} channel assigned.
+
+
+.Procedure: Adding a Node to a Cluster
+. In the {productname} {webui}, navigate to menu:Clusters[Overview] and click btn:[Join Node].
+. Select the node to add from a list of available nodes.
```

Comment:
multiple nodes can be selected, so I would say `node or nodes`

Reply:
Fixed